sys/targets/targets.go

```diff
@@ -205,7 +205,7 @@ var List = map[string]map[string]*Target{
 		TestArch32ForkShmem: {
 			PtrSize:  4,
 			PageSize: 4 << 10,
-			CFlags:   []string{"-m32", "-static"},
+			CFlags:   []string{"-m32", "-static-pie"},
```

Comment:
It may be useful to leave one of the test targets as -static to get more coverage.